hyppo/kgof/data.py

```diff
@@ -0,0 +1,845 @@
+"""
```

Comment:
My test statistic calculation and the module as a whole is pretty dependent on the structure of this Data object (as well as Kernel and Density) given I needed to easily be able to pull datasources. There's overlapping functionality with sims that exist in hyppo.tools, but is there any way these three files can stay under perhaps a more specific name? It would require pretty extensive rewrites to rip these out and am worried it'll compromise some of the functionality.

Reply:
ok, make a comment saying this in the file for now, and then for the beginning of next semester you can rework the code so it doesn't rely on this object. Inputs in hyppo are consistently nxp numpy arrays and would require a heavy API rewrite of the rest of the package if this is the input

Reply:
Sounds good. This is what I was going to suggest as well. I'll spend next semester reducing dependencies and making it more in-line with the existing API structure. Thanks!